tests/FSi/DoctrineExtensions/Tests/Uploadable/Fixture/Inheritance/Employee.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * (c) FSi sp. z o.o. <info@fsi.pl>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FSi\DoctrineExtensions\Tests\Uploadable\Fixture\Inheritance;
+
+use Doctrine\ORM\Mapping as ORM;
+use FSi\DoctrineExtensions\Uploadable\File;
+use FSi\DoctrineExtensions\Uploadable\Mapping\Annotation as FSi;
+use SplFileInfo;
+
+/**
+ * @ORM\Entity
+ */
+class Employee extends Person
+{
+    /**
+     * @ORM\Column(nullable=true)
+     * @FSi\Uploadable(targetField="file")
+     */
+    private $filePath;
```

Comment:
I think some problem shows up when this field (and/or `$file`) is defined in parent class